Userland/Applications/3DFileViewer/main.cpp

```diff
@@ -226,8 +246,21 @@ bool GLContextWidget::load(const String& filename)
     builder.append(filename.split('.').at(0));
     builder.append(".bmp");
 
+    String texture_path = Core::File::absolute_path(builder.string_view());
+
     // Attempt to open the texture file from disk
-    auto texture_image = Gfx::Bitmap::try_load_from_file(builder.string_view());
+    RefPtr<Gfx::Bitmap> texture_image;
+    if (Core::File::exists(texture_path)) {
```

Comment:
Why try to open this using `Core::File` if we're using FSAC instead? This just requires us to have different methods to load from name / fd+name, etc.

Reply:
This duplication exists to let the application load the teapot on start-up in conjunction with the unveils. I think this makes sense unless the teapot on start-up is removed.